bakery/pipeline-with-service.yml

```diff
@@ -52,6 +52,7 @@ resources:
     type: workflow-events
     source:
       api_root: https://cc1.cnx.org/api
+      status_id: 0
```

Comment:
There should be no `status_id == 0`. I don't see how this would work.

Reply:
This addresses the `KeyError`. 
Note of this made here: 
https://github.com/openstax/cnx/issues/764#issuecomment-552657589

(I imagine it's in some kind of holding state until the actual put occurs to change the status.)

but ultimately we would need to update the `check` part of the resource.

Reply:
I think it's worth having a discussion about how concourse resources are used. For example, should `status_id` be optional if someone wants to intentionally select nothing, because they only want to put and not get the resource? When putting has nothing to do with versions of the resource (i.e. the check params), should we always make a separate resource of the same type for putting? I think there are arguments for a variety of conventions since parameters can sometimes be in, out exclusive, and sometimes not, depending on the situation.

Reply:
I've captured @tomjw64 thoughts above, in issue #781